deviceModeInit.js

```diff
@@ -117,16 +115,143 @@ var rudderTracking = (function () {
         const needToUpdateCart = checkCartNeedsToBeUpdated(cart);
         if (needToUpdateCart) {
           updateCartAttribute().then((cart) => {
-            sendIdentifierToRudderWebhook(cart); // sending rudderIdentifier periodically after this
+            sendIdentifierToRudderWebhook(cart);
           });
           console.log("Successfully updated cart");
```

Comment:
Should we use console.error here?

Reply:
Since it is not an error we can use `console.debug`

Reply:
awesome!

Reply:
I meant for those where we are using .log for logging error messages. For now, this will suffice

Reply:
Changed the console statements